platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/cache/ConcurrentArray.java

```diff
@@ -272,43 +273,26 @@ void add(T element) {
     }
 
     /**
-     * Get a sorted stream of all elements. The comparator will be used to sort the elements.
-     * If the comparator is null, then the natural ordering of the elements will be used.
-     * This method can only be called on immutable instances. It may be called concurrently.
+     * Get a stream of all elements in this concurrent array.
      *
-     * @param comparator
-     * 		The comparator to use to sort the elements. May be null, in which
-     * 		case natural ordering is used.
-     * @return A non-null Stream over all elements.
+     * <p>This method can only be called on immutable instances. It may be called concurrently.
+     *
+     * @return A non-null stream over all elements
      * @throws IllegalStateException
-     * 		If this instance is not immutable.
+     * 		If this instance is not immutable
      */
-    @SuppressWarnings("unchecked")
-    Stream<T> sortedStream(Comparator<T> comparator) {
+    Stream<T> stream() {
         if (!immutable.get()) {
-            throw new IllegalStateException("You can not call sortedStream on a mutable ConcurrentArray");
+            throw new IllegalStateException("You can not call stream() on a mutable ConcurrentArray");
         }
 
-        // A quick exit: if there are no elements, then return an empty stream.
+        // A quick exit: if there are no elements, then return an empty stream
         final int numberOfElements = this.elementCount.get();
         if (numberOfElements == 0) {
-            return Stream.empty();
+            return Stream.of();
```

Comment:
Just for my understanding. Is there any particular difference or it's as shorter form of the same thing?

Reply:
This change is not intentional. Let me revert it back to `empty()`